### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by boyuanx** [sannidhyaroy/Soduto#19] [General]
Hey Sannidhya, I've been using this for a while and it's helped me a lot. I appreciate what you've been doing and would like to make some donations. How can I get in touch? Thanks!

**Comment by sannidhyaroy** [comment]
I'm truly glad to hear that the app has been useful to you. Your kind words mean a lot.

As a student and a hobbyist developer, I've been working on Soduto because I find it personally valuable and fulfilling. While your donation offer is very generous, I'd like to suggest redirecting any support towards the original Soduto developers. They laid the foundation for this fantastic app, and they deserve credit for their work.

Thank you again for your kind offer and for using Soduto.

**Reply by boyuanx** [reply]
Unfortunately I don't think they are planning to resume development... If they are still active then I would have donated for sure. At this point you've done just as much if not even more than the original developer. I used to be a student and hobbyist developer as well so I know how it is. If anything, at least you can notarize the app with the donations.
